flexget/plugins/plugin_transmission.py

```diff
@@ -298,6 +297,13 @@ def on_task_output(self, task, config):
         if task.accepted:
             self.add_to_transmission(self.client, task, config)
 
+    def _decode_if_bytes(self, data):
+        """Transmission doesn't like encoded byte strings (Py3) so decode"""
+        if type(data) == bytes:
```

Comment:
Isn't `isinstance` the preferred way of checking for types since it properly supports inheritance?